Rakefile

```diff
@@ -78,6 +78,16 @@ namespace :build do
     end
   end
 
+  task :shared do
```

Comment:
Apps and shared are very similar. Might be some opportunity to share code, especially since eventually I'd like to get some of the JS in dashboard to also go through browserify, which means it will also likely need a step like this.